app/views/admin/daily_questions/_form.html.haml

```diff
@@ -0,0 +1,17 @@
+.field
+  %h4= f.label 'Question'
+  = f.text_field :question_id, class: 'form-control', id: 'Question', required: true
+.field
+  %h4= f.label 'Programmée le'
+  = f.text_field :scheduled_at, :id => "datepicker", value: '', class: 'form-control'
+  :javascript
+    $('#datepicker').datepicker({
+      format: 'dd-mm-yyyy',
+      language: "fr",
+      todayBtn   : 'linked',
+      todayHighlight: true,
+    });
+  %h4= f.label 'Nombre de réponses obtenues'
+  = f.text_field :answer_counter, class: 'form-control', id: 'Nombre de réponses obtenues', required: true, autofocus: true
```

Comment:
I don't understand this line:

- why a text_field to display to amount of anwsers?
- isn't this ID a bit strange? (with spaces)